lms/djangoapps/course_wiki/course_app.py

```diff
@@ -0,0 +1,64 @@
+"""Module with the course app configuration for the Wiki."""
+from typing import Dict, Optional
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import ugettext_noop as _
+from edx_toggles.toggles import SettingToggle
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.course_apps.plugin import CourseApp
+
+User = get_user_model()
+
+
+class WikiCourseApp(CourseApp):
+    """
+    Course app for the Wiki.
+    """
+
+    app_id = "wiki"
+    name = _("Wiki")
+    description = _("Enable learners to access, and collaborate on information about your course.")
+    default_enabled = SettingToggle(name="WIKI_ENABLED", default=True).is_enabled()
+
+    @classmethod
+    def is_available(cls, course_key: CourseKey) -> bool:  # pylint: disable=unused-argument
+        """
+        Returns if the app is available for the course.
+
+        The wiki is available for all courses or none of them depending on the a Django setting.
+        """
+        return cls.default_enabled
+
+    @classmethod
+    def is_enabled(cls, course_key: CourseKey) -> bool:  # pylint: disable=unused-argument
+        """
+        Returns if the wiki is available for the course.
+
+        The wiki currently cannot be enabled or disabled on a per-course basis.
+        """
+        return cls.default_enabled
+
+    @classmethod
+    def set_enabled(
+        cls, course_key: CourseKey, user: User, enabled: bool  # pylint: disable=unused-argument
+    ) -> bool:
+        """
+        The wiki cannot be enabled or disabled.
+        """
+        # Currently, you cannot enable/disable wiki via the API
+        return cls.default_enabled
+
+    @classmethod
+    def get_allowed_operations(
+        cls, course_key: CourseKey, user: Optional[User] = None
+    ) -> Dict[str, bool]:  # pylint: disable=unused-argument
+        """
+        Returns the operations you can perform on the wiki.
+        """
+        return {
```

Comment:
To indicate no operations can be performed on the wiki can we return None? Would look more clean